MathPortal/MathPortal/Source/Custom/KeyboardButton.swift

```diff
@@ -0,0 +1,64 @@
+//
+//  KeyboardButton.swift
+//  MathPortal
+//
+//  Created by Petra Čačkov on 27/10/2019.
+//  Copyright © 2019 Petra Čačkov. All rights reserved.
+//
+
+import UIKit
+
+class KeyboardButton: UIButton {
```

Comment:
In such cases it is better to subclass an `UIView` instead of a `UIButton`. This is a bit dangerous but leave it for now I guess.